06plusMinus.rb

```diff
@@ -0,0 +1,22 @@
+def plusMinus(arr)
+    puts(arr)
+    positive = 0
+    negative = 0
+    zero = 0
+    arr.each do |number|
+        if arr[number] > 0
```

Comment:
Ruby accepts if statements after the code. Eg. `<do this> if <this is true>`.
Your code is fine but you can try using the above to get used to it.
Eg.
```
arr.each do |number|
  positive += 1 if arr[number] > 0
  negative += 1 if arr[number] < 0
  zero += 1 if arr[number] == 0
end
```